onmt/opts.py

```diff
@@ -643,30 +644,42 @@ def _add_decoding_opts(parser):
                    "target token. If it is not provided (or the identified "
                    "source token does not exist in the table), then it "
                    "will copy the source token.")
+    group.add('--ban_unk_token', '-ban_unk_token',
+              action="store_true",
+              help="Prevent unk token use by setting unk proba to 0")
```

Comment:
Maybe "generation" instead of "use", the unk token will still be used on source side if a token is not in vocab.